templates/shared/_insights_news_strip.html

```diff
@@ -9,40 +9,58 @@
-          </div>
-        {% endfor %}
+{% if insights_feed is False %}
+  <section class="p-strip is-deep is-bordered">
+    <div class="row">
+      <div class="col-8">
+        <h2 class="p-link--external p-heading--insights__title">
+          <a href="https://insights.ubuntu.com/" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'insights', 'eventAction' : 'clicks insights feed link - feed error', 'eventLabel' : '{{ gtm_event_label }}', 'eventValue' : undefined });">
+            Latest news from Insights
+          </a>
+        </h2>
+        <p>The latest Ubuntu news is available on the <a href="https://insights.ubuntu.com/" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'insights', 'eventAction' : 'clicks insights feed message link - feed error', 'eventLabel' : '{{ gtm_event_label }}', 'eventValue' : undefined });" class="p-link--external">Insights website</a>.</p>
+        <div class="p-notification--negative">
+          <p class="p-notification__response"><span class="p-notification__status">Error:</span>The live news feed failed to load. Please <a href="https://github.com/canonical-websites/www.ubuntu.com/issues/new" class="external">report this bug</a> and our team will fix the problem as soon as possible.</p>
+        </div>
       </div>
     </div>
-
-    {# spotlight heading #}
-    {% if spotlight_feed %}
-      <div class="col-3 p-divider__block">
+  </section>
+{% else %}
+  <section class="{% if strip_classes %}{{strip_classes}}{% else %}p-strip is-deep is-bordered{% endif%}">
+    <div class="row {% if spotlight_feed %}p-divider{% endif %}">
+      <div class="{% if spotlight_feed %}col-9 p-divider__block{% else %}col-12{% endif %}">
+        {# insights heading #}
         <h2 class="p-link--external p-heading--insights__title">
-          <a href="https://insights.ubuntu.com/tag/spotlight/" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'insights', 'eventAction' : '{{ gtm_event_label }} spotlight feed link', 'eventLabel' : '{{ gtm_event_label }}', 'eventValue' : undefined });">
-            Spotlight
+          <a href="https://insights.ubuntu.com/" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'insights', 'eventAction' : 'clicks insights feed link', 'eventLabel' : '{{ gtm_event_label }}', 'eventValue' : undefined });">
+            Latest news from Insights
           </a>
         </h2>
-        {% for item in spotlight_feed %}
-          <h3 class="p-heading--four u-less-margin--bottom"><a href="{{ item.link }}" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'insights', 'eventAction' : '{{ gtm_event_label }} spotlight article', 'eventLabel' : '{{ item.title|escapejs }}', 'eventValue' : '{{ item.link }}' });">{{ item.title }}</a></h3>
-          <p><time pubdate datetime="{{ item.updated }}">{{ item.updated_datetime|date:"j F Y" }}</time></p>
-        {% endfor %}
+
+        {# insights news items #}
```

Comment:
Couldn't the `if insights_feed` conditional be here? So we don't need to duplicate the whole `<section>`?